packages/components/src/Actions/ActionButton/ActionButton.component.js

```diff
@@ -172,6 +172,7 @@ export function ActionButton(props) {
 	if (!inProgress && overlayComponent) {
 		btn = (
 			<OverlayTrigger
+				onClick={rClick}
```

Comment:
i don't get this part, where would be used the button onClick in the overlay ?

Reply:
Before to open the overlay, i need to save the current index of the line which is open.